NuGet.config

```diff
@@ -13,6 +13,7 @@
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
     <add key="vssdk" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk/nuget/v3/index.json" />
     <add key="vs-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json" />
+    <add key="Nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
This is going to block merge, any new dependencies needed for this should be listed s that we can ask folks to add them to the internal feeds (which are public but just 'blessed' versions of packages to prevent version-substitution attacks from NuGet.org)

Reply:
Have removed now - we don't need it as we don't reference OpenTelemetry stuff anymore.